lib/locale-strings.js

```diff
@@ -0,0 +1,15 @@
+const _ = require('sdk/l10n').get;
+
+module.exports = {
+  errorMsg: _('error_msg'),
+  errorLink: _('error_link'),
+  loadingMsg: _('loading_msg'),
```

Comment:
You could use interpolation here to insert a variable that you'd then replace with the domain name in the React code, like: `loadingMsg: _('loading_msg', '$domain')`. Then the React code could just do a simple string replace on '$domain'.